README

```diff
@@ -36,13 +40,38 @@ must be switched ON. One must manually give the explicit path to magma.h in
 the file blas_magma.cu.
 
 
+Overview:
+
+The QKXTM (Quda Kepler and Twisted Mass) calculation routines are appended
+to the interface_quda.cpp file of mainline QUDA. This is so we can use the
+internal gauge field structures directly. The class definitions and header
+files of auxiliary functions are contained in files prepended like so,
+
+   QKXTM_<some file>.suffix
+
+There is a single .cu file for kernels, a 'code_pieces' directory in lib/,
+a single qudaQKXTM_Kepler_utils.cpp and .h file for containers and general 
+functions, and a suite of qudaQKXTM_<CLASS>_Kepler.cpp files under a single
+header qudaQKXTM_Kepler.h. These <CLASS> .cpp files are NOT individually
+compiled, rather included in interface_quda.cpp as,
+
+... [end interface_quda.cpp code]
+
+#include <qudaQKXTM_Field_Kepler.cpp
+...
+#include <qudaQKXTM_Deflation_Kepler.cpp
+
+... [begin QKXTM calculation functions]
+
 Known Issues:
 
 One must pass gauge fields to the QKXTM executables in IDLG lime format. It
 expects a separate smeared gauge field for source construction in the
 hadronic 2pt3pt functions. QUDA has gauge smearing functionality, which
 should be utilised.
-
+It would be desirable to completely separate out the QKXTM files from QUDA
+so that a `plug-in` can be installed alongside any suitable compiled copy
+of QUDA.
 
 Authors:
 
```

Comment:
As it is stated here it might imply that Kyriakos and I have developed the whole QUDA library. I would suggest that we are a bit more specific here. We could clearly state that Kyriakos and I developed the QKXTM code parts, e.g.:

The QKXTM code pieces were developed by,

Kyriakos Hadjiyiannakou
Christos Kallidonis